pkgs/development/python-modules/verilogae/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, fetchurl, buildPythonPackage, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "verilogae";
+  version = "1.0.0";
+  src = fetchurl {
+    url =
+      "https://files.pythonhosted.org/packages/aa/a6/9daea00745844faba44c2917c66838adfc315269f38518ecca49e6eb5fb5/verilogae-1.0.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
+    sha256 = "QsEVq/4c44xCkovOhXOj8Zh6rp4Ipq+NGjbTW25Fd6E=";
+  };
```

Comment:
Any reasons for not getting this via `fetchPypi` or from the source repo directly?

Reply:
As previously mentioned there were issues using fetchPypi I can try with source repo if that's preferred.